doc/source/management/enforce_engine.rst

```diff
@@ -0,0 +1,55 @@
+.. _enforce_engine:
+
+========================
+Enforcing Storage Engine
+========================
+
+|Percona Server| has implemented variable which can be used for enforcing the
```

Comment:
"has implemented the `enforce_storage_engine`" variable? The name isn't mentioned at all until the bottom of this page.

Reply:
Fixed. Added the variable name to the first sentence.